bwmenu

```diff
@@ -285,10 +326,10 @@ show_copy_notification() {
   local body=""
   local extra_options=()
 
-  title="<b>$(echo "$1" | jq -r '.name')</b> copied"
```

Comment:
Not a fan of the name **in bold**? :stuck_out_tongue:

Reply:
Not sure what's behind my `notify-send` binary, but it doesn't parse this. :duck: 
I didn't mean to include this, oops.